docs/integrate_gcp_using_supported_system.md

```diff
@@ -0,0 +1,176 @@
+
+If you already have an account configured with these permissions you can skip the next section
+
+#### Creating a Service Account using the GCP Console
+This section convers creating a Google Cloud Service Account with the required permissions to integrate Lacework at an Organization level.
+![Service Account Project Owner](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_service_account_create_complete_1024px.gif)
+
+1. Login to the Google Cloud Console
+2. Select the Project created for Lacework Resources
+3. Click on the **Navigation Menu** and select **IAM & Admin** -> **Service Accounts**
+4. Click **CREATE SERVICE ACCOUNT**
+5. Give the service account a name (i.e. terraform-provisioning) and optionally a description, and click **CREATE**
+6. Under the section **"Grant this service account access to project"** give the service account **"Owner"** permissions to the project
+7. Click **SAVE**
+
+#### Add Service Account to Organization with required permissions
+This section covers adding the service account as a member of the organziation and configure the required organization permisions for Terraform to configure the organization
+
+1. Select the organization you are going to integrate with Lacework, select **IAM** from the Navigation Menu, and then click the **+ADD** button to add a member or role to the organization
+2. Search for the service account then add permissions for **Organization Administrator** and **Logs Configuration Writer**
+3. Click **SAVE**
+
+![Service Account Org Permissions](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_service_account_org_permissions_1024px.gif)
+
+#### Create Service Account Key
+This section covers creating a service key and downloading to the local system as a `JSON` file
+
+![Create Key](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_create_key_1024px.gif)
+
+1. Login to the Google Cloud Console
+2. Select the Project created for Lacework Resources
+3. Click on the **Navigation Menu** and select **IAM & Admin** -> **Service Accounts**
+4. Click on the Actions menu next to the service account
+5. Click **Create Key**
+6. Select JSON for the format of the key
+7. Click **Create** to download the key locally
+
+### Run Terraform to integrate GCP at Organization Level
+This section covers executing Terraform. It is recommended you use a developer's code editor such as [VSCode](https://code.visualstudio.com/download), [Atom](https://atom.io/), or [Sublime](https://www.sublimetext.com/3). You will also need a terminal to run `terraform` commands such as `bash`, `zsh`, `powershell` or `cmd`
+![Terraform Code](https://techally-artifacts.s3-us-west-2.amazonaws.com/terraform-module-docs/gcp_org_terraform_init_1024px.gif)
+1. Open an EDITOR of choice and create a new file called `main.tf`
+2. Copy/paste the following code into the `main.tf` and save the file
+```hcl
+provider "google" {
```

Comment:
before presenting this code, we need to add this important note:
>**IMPORTANT:** We use the `master` branch in source just as an example. In your code, **do NOT pin to master**
because there may be breaking changes between releases. Instead we recommend to pin to the release tag (e.g.
`?ref=tags/v0.1.0`) of one of our [latest releases](https://github.com/lacework/terraform-provisioning/releases).